js/hview.js

```diff
@@ -585,7 +585,20 @@ $(function () {
         elWrap.appendChild(xprofileProtoType.cloneNode())
     
         const el = labelProtoType.cloneNode()
-        container.appendChild(el)
+        let insertBeforeEl = null
+        if (node.insertBeforeName != null) {
```

Comment:
if two consecutive nodes are added, insertBeforeName for the first node (which would point to the future second node) may not exist yet.

Reply:
I will resolve this in the next revision. I could either account for this in the Diff.js calculation, or ensure that the order of node insertions prevents this from happening.